AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_EntityReferenceAdded.py

```diff
@@ -60,13 +81,21 @@ def AtomEditorComponents_EntityReference_AddedToEntity():
     2) Add Entity Reference component to Entity Reference entity.
     3) UNDO the entity creation and component addition.
     4) REDO the entity creation and component addition.
-    5) Enter/Exit game mode.
-    6) Test IsHidden.
-    7) Test IsVisible.
-    8) Delete Entity Reference entity.
-    9) UNDO deletion.
-    10) REDO deletion.
-    11) Look for errors.
+    5) 'EntityIdReferences' is a container property
+    6) Append item to 'EntityIdReferences'
+    7) Add item to 'EntityIdReferences'
+    8) Update item in 'EntityIdReferences'
+    9) Remove item from 'EntityIdReferences'
+    10) Rest the container property then put one entity reference back for further tests
+    11) Remove component
+    12) Undo component remove
```

Comment:
nitpick: can we capitalize UNDO here also since the others are capitalized? not a big deal either way, but felt worth noting

Reply:
UNDO